### PR TITLE
Merge post-1.46 in its entirety

### PR DESCRIPTION
**Pull request by smblott-github**
`post-1.46` is the branch that has been collecting less well bedded in features from around the time of the 1.46/9 release.

It seems stable.  Suggest we merge it in its entirety.

What does everybody else think?

Mention: @mrmr1993, @philc.

Edit:  Most of what this contains is [here](https://github.com/philc/vimium/issues?q=label%3Ain-post-1.46+is%3Aclosed).

### CODE REVIEW

**Comment by philc** [comment]
Playing with it now.

**Comment by philc** [comment]
Overall it looks and feels good to me. I played with the vomnibar and link hints.

File: content_scripts/vimium_frontend.coffee [hunk 1]
```diff
@@ -436,15 +454,16 @@ onKeydown = (event) ->
...
-      exitInsertMode()
-      DomUtils.suppressEvent event
-      KeydownEvents.push event
+      # NOTE(smblott, 2014/12/22) Including embeds for .blur() etc. here is experimental.  It appears to be
+      # the right thing to do for most common use cases.  However, it could also cripple flash-based sites and
+      # games.  See discussion in #1211 and #1194.
+      event.srcElement.blur()
+    exitInsertMode()
+    DomUtils.suppressEvent event
+    handledKeydownEvents.push event
```
Comment (z0rch) [inline content_scripts/vimium_frontend.coffee]: `handledKeydownEvents` is undefined here, it should say `KeydownEvents`.

Comment (smblott-github) [inline content_scripts/vimium_frontend.coffee]: Thanks: fc2201b996e47ca06090e10e4ebfcd9f4b345fde.

**Comment by smblott-github** [comment]
Merging.  Rationale:
- `post-1.46` contains lots of good stuff.
- It seems to be stable.
- It's quite some weeks since 1.49, and we're not seeing big problems (so probably no quick release needed).
- We need to move forward.

**Pull request merged**